src/mixins/nft.js

```diff
@@ -656,8 +656,9 @@ export default {
       this.NFTHistory = this.nftIsWritingNFT
         ? populateGrantEvent(latestBatchEvents, dbEventMap)
         : latestBatchEvents;
+      const uniqueAddresses = getUniqueAddressesFromEvent(this.NFTHistory);
       this.lazyGetUserInfoByAddresses(
-        getUniqueAddressesFromEvent(this.NFTHistory)
+        getAllUserInfo ? uniqueAddresses : uniqueAddresses.slice(0, 10)
```

Comment:
Please name `10` as some const, and preferably as a optional param in this function